apps/settings/elements/call_fdn_settings.html

```diff
@@ -41,7 +41,6 @@ <h1 data-l10n-id="fdnSettings-header"></h1>
     </div>
 
     <script src="js/simcard_dialog.js"></script>
```

Comment:
Hmm it seems that we forgot to remove this line last time, let me remove it here.